packages/compiler-core/src/transforms/hoistStatic.ts

```diff
@@ -151,7 +151,7 @@ function isStaticNode(
       })
     default:
       if (__DEV__) {
-        const exhaustiveCheck: never = node
```

Comment:
Looks like we need `never` annotation here as it is meant to check that the code covers all `node` types in `case` statements. i.e. If there are missing check for `NodeTypes`, it produces compile error.

Reply:
`node` is already never 🤔 

![image](https://user-images.githubusercontent.com/4620458/66898126-63f99000-eff0-11e9-88c3-68ed68023304.png)

Reply:
I mean if we add `NodeTypes` in the future the `node` will be other than `never` before we cover the `NodeTypes` in the case statements. In that case this line helps us to detect it and make sure to cover all `NodeTypes` in this switch statement by producing compile error.

Reply:
makes sense, thanks 👍